backend/src/main/java/org/sefglobal/core/controller/CertificateController.java

```diff
@@ -0,0 +1,40 @@
+package org.sefglobal.core.controller;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.model.Certificate;
+import org.sefglobal.core.repository.CertificateRepository;
+import org.sefglobal.core.service.CertificateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/fellowship")
+public class CertificateController {
+
+    @Autowired
+    private CertificateService service;
```

Comment:
change variable name to `certificateService`

Reply:
Sure!